### ISSUE TITLE
JSON is not defined - ignoring global configuration

### ISSUE THREAD

**Issue by benmarten** [gruntjs/grunt-contrib-jshint#203]
Using version 0.11
I have no .jshintrc, only this in my Gruntfile.js

		jshint: {
			options: {
				jshintrc: true,
				JSON: true
			},
			files: ['index.js', 'Gruntfile.js', 'lib/*.js', 'test/*.js']
		}

Erorr:

     29 |   test = new Test(JSON.stringify(text), function(done) {
                            ^ 'JSON' is not defined.

When downgrading to 0.10 it works. What's the issue?

**Comment by benmarten** [comment]
any thoughts?

**Comment by OmerHerera** [comment]
When upgrading to version: 0.11.2 its working.
But I'm wandering why this should be in the global configuration , in my opinion this should be always true.

**Comment by benmarten** [comment]
yep, works with latest version. even without `JSON: true`. so i think it works by default again, doesn't it?

**Comment by OmerHerera** [comment]
No, its not working I need to set: ```JSON: true```

**Comment by furzeface** [comment]
Should it be:

```
jshint: {
        options: {
            jshintrc: true,
                global: {
                    JSON: true
                }
        },
        files: ['index.js', 'Gruntfile.js', 'lib/*.js', 'test/*.js']
    }
```

:question:

**Comment by brianjmiller** [comment]
Appears that with newer jshint it will depend on what setting you have for `es3` which is why it starts to show up around 0.11.0 because that is when jshint 2.6.0 started to land which includes https://github.com/jshint/jshint/commit/da52aa096a1a5c3c40e731c9bcceb038e410deeb which i think causes this. Either way @furzeface is correct, moving `JSON: true` into `options.global` should fix it, as does switching `es3` to `false`. (`JSON` wasn't in es3 in various IE versions.)
